go.mod

```diff
@@ -80,8 +80,10 @@ require (
 
 replace (
 	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20210409032903-31b989a197eb // Use OpenShift fork
+	github.com/openshift/hive/apis => github.com/dgoodwin/hive/apis v0.0.0-20210426140401-97292266a297
```

Comment:
@dgoodwin this point to your branch. Should I change to https://github.com/openshift/hive/tree/cluster-installs ?

Reply:
When will it be merged to master?

Reply:
We're working on it, should be within a couple days.